enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -0,0 +1,109 @@
+---
+title: update-blocker-lifecycle
+authors:
+  - "@wking"
+reviewers:
+  - "@LalatenduMohanty"
+approvers:
+  - TBD
+creation-date: 2020-09-11
+last-updated: 2021-03-30
+status: implementable
+---
+
+# Update-blocker Lifecycle
+
+We occasionally have bugs which impact update success or the stability of the target release.
+When that happens, we protect users by [removing the impacted updates from our update recommendations][graph-data-block].
```

Comment:
IMO we over use this mechanism, and the rational / reason for over using it is because we can't tailor the updates to specific accounts or classes of accounts. 

IE: https://issues.redhat.com/browse/OTA-267 & https://issues.redhat.com/browse/OTA-191